config/environments/development.rb

```diff
@@ -41,12 +52,19 @@
   # Print deprecation notices to the Rails logger.
   config.active_support.deprecation = :log
 
+  # Raise exceptions for disallowed deprecations.
+  config.active_support.disallowed_deprecation = :raise
+
+  # Tell Active Support which deprecation messages to disallow.
+  config.active_support.disallowed_deprecation_warnings = []
+
   # Raise an error on page load if there are pending migrations.
   config.active_record.migration_error = :page_load
 
+  # Highlight code that triggered database queries in logs.
+  config.active_record.verbose_query_logs = true
```

Comment:
What it sounds like! Adds verbosity to DB logs - https://edgeguides.rubyonrails.org/configuring.html#config-active-record-verbose-query-logs